docs/nginx.md

```diff
@@ -128,14 +135,15 @@ nginx:
   links:
     - registry:registry
   volumes:
-    - `pwd`/auth/:/etc/nginx/conf.d
+    - ./auth:/etc/nginx/conf.d
+    - ./auth/nginx.conf:/etc/nginx/nginx.conf:ro
 
 registry:
   image: registry:2
   ports:
     - 127.0.0.1:5000:5000
   volumes:
-    - `pwd`/data:/var/lib/registry
+    - ./data:/var/lib/registry
```

Comment:
I see your point. I had copy/pasted into an edit as opposed to using <<EOF , and hence ended up with the pwd inside the yaml file.